test/util/load_sku_packs.py

```diff
@@ -35,40 +39,38 @@ def test01_preload_sku_packs(self):
                     subprocess.call("cd on-skupack; ./build-package.bash "
                                     + skus + " " + skus + " >/dev/null 2>&1", shell=True)
             break
```

Comment:
Can you break the "build sku packs" subsection into a separate test too?  When using the script, I'll pull down the sku packs repo, modify some data in an existing sku pack or create a new one, then call your build code chunk.    Then I can keep calling the build routine....without downloading again.